src/CoreWCF.Templates/src/templates/CoreWCFService/.template.config/dotnetcli.host.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
add the vs support file like here https://github.com/dotnet/aspnetcore/blob/main/src/ProjectTemplates/Web.ProjectTemplates/content/EmptyWeb-CSharp/.template.config/ide.host.json

Reply:
![image](https://user-images.githubusercontent.com/681739/165898921-f0795a80-d6de-4bf5-af1f-8d14c2180ba3.png)

Reply:
![image](https://user-images.githubusercontent.com/681739/165899131-dac9a0fd-3d8b-4989-9ba9-67f3fc948c22.png)